### DISCUSSION TITLE
Multiple Instances Conflicting with Traefik 2.3

### DISCUSSION THREAD

**Discussion by cilise** [felddy/foundryvtt-docker#179] [Q&A] [answered]
I purchased a second license so I could run to games side by side.  I run docker/compose in ubuntu 18.04 lts.  I copy/pasta'd my docker compose config and made the port changes on the new container to a different port.  When it started, both instances 404'd.  So I looked around and didn't notice an ENV value to force a port change, I discovered Q/A post #134 where someone else had a similar issue.  
I added the following to both my configs:

ports:
  - "newport:30000" 
(since I'm reverse proxy'ing I shouldnt need to expose these but i wanted to test anyways"

Then I updated my traefik label - "traefik.http.services.-svc.loadbalancer.server.port=newportnumber"   

This still doesn't make a difference.  I'm sure I'm missing something easy, but I've been staring at this too long and need a new set of eyes on it.  

-edit:  I thought I fixed this since I realized I'd forgot some update label definitions from my copy/paste.  When I corrected this, both my urls resolved to the same destination even though they are different containers.

**Comment by cilise** [comment, marked as answer]
So I found my problem.  I had missed a single label update.  Guess I just needed to walk away from it for a few.  We can mark this closed.

**Reply by felddy** [reply]
Glad you figured it out!
